diff --git a/dist/chunks/global.d.MAmajcmJ.d.ts b/dist/chunks/global.d.MAmajcmJ.d.ts
index 3fa6a6e96fda091a6270c16fd076debccd2ec4d0..7e0e8b34c6b05874be5b506b226a4784765933dd 100644
--- a/dist/chunks/global.d.MAmajcmJ.d.ts
+++ b/dist/chunks/global.d.MAmajcmJ.d.ts
@@ -34,9 +34,6 @@ declare global {
 		interface Assertion {
 			containSubset: (expected: any) => Assertion;
 		}
-		interface Assert {
-			containSubset: (val: any, exp: any, msg?: string) => void;
-		}
 	}
 }
 interface SnapshotMatcher<T> {
